dll/win32/gdiplus/font.c

```diff
@@ -638,7 +638,15 @@ static INT CALLBACK is_font_installed_proc(const LOGFONTW *elf,
     if (type & RASTER_FONTTYPE)
         return 1;
 
+#ifdef __REACTOS__
```

Comment:
May be worth sending to Wine at some point...

Reply:
Don't use my name in posting to Wine. Wine doesn't like me.

Reply:
This change was wrong. Reverted.